src/cls/IPM/Storage/MappingReference.cls

```diff
@@ -10,7 +10,7 @@ Property Name As %String(MAXLEN = 2048, XMLPROJECTION = "attribute") [ Required
 /// Name index is unique per namespace.
 Index ResourceName On (Name, Source);
 
-Relationship Module As %IPM.Storage.Module(XMLPROJECTION = "NONE") [ Cardinality = parent, Inverse = Mappings ];
+Relationship Module As %IPM.Storage.Module(XMLPROJECTION = "NONE") [ Cardinality = parent, Inverse = Mappings, SqlFieldName = _MODULE ];
```

Comment:
@isc-tleavitt For consistency, would make sense to call this `ModuleItem` to match the `SqlTableName` of `%IPM.Storage.Module`

Reply:
Fair point; I'll do this everywhere I'd updated to _MODULE